package.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+project=$(cat package.json | jq -r .name)
+version=$(cat package.json | jq -r .version)
+iteration="$TRAVIS_BUILD_NUMBER.git$(git rev-parse --short HEAD)"
```

Comment:
Travis_BUILD_NUMBER should have a default value for when running locally as testing